src/Symfony/Bridge/Doctrine/ContainerAwareEventManager.php

```diff
@@ -189,7 +189,7 @@ private function initializeSubscribers(): void
                 $listener = $this->container->get($listener);
             }
             // throw new \InvalidArgumentException(sprintf('Using Doctrine subscriber "%s" is not allowed, declare it as a listener instead.', \is_object($listener) ? $listener::class : $listener));
-            trigger_deprecation('symfony/doctrine-bridge', '6.3', 'Using Doctrine subscribers as services is deprecated, declare listeners instead');
+            trigger_deprecation('symfony/doctrine-bridge', '6.3', 'Using Doctrine subscriber "%s" as service is deprecated, declare it as a listener instead', \is_object($listener) ? $listener::class : $listener);
```

Comment:
As pretty much everything is a service in Symfony, the message still feels confusing. 
What do you think about:
```suggestion
            trigger_deprecation('symfony/doctrine-bridge', '6.3', 'Implementing "%s" in "%s" is deprecated, declare it as a listener instead.', EventSubscriber::class, \is_object($listener) ? $listener::class : $listener);
```

Reply:
You're right, PR is now updated

Reply:
get_debug_type() instead of::class